sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -96,6 +96,7 @@ class PlanResolutionSuite extends AnalysisTest {
           throw new CatalogNotFoundException(s"No such catalog: $name")
       }
     })
+    when(manager.currentCatalog).thenReturn(testCat)
     when(manager.defaultCatalog).thenReturn(Some(testCat))
```

Comment:
I will clean this up when I restrict the access to default/session catalog in `CatalogManager`. This is a temporary fix to check if all the tests are passing.